src/omotes_sdk/omotes_interface.py

```diff
@@ -319,6 +329,8 @@ def submit_job(
         :param auto_disconnect_on_result: Remove/disconnect from all queues pertaining to this job
             once the result is received and handled without exceptions through
             `callback_on_finished`.
+        :param job_reference: An optional reference to the submitted job which is used in the
+            name of the output ESDL as well as in internal logging of OMOTES.
```

Comment:
Very nice to have it in the internal logging as well. This is to be implemented?

Reply:
Yes! Originally meant for a more bare implementation to check if the implementation fits everyone. Logging introduced here: https://github.com/Project-OMOTES/omotes-sdk-python/pull/77/commits/e67013198a816de4e2a7c7f06abc21b132b00a0f